src/common/BaseLightAccount.sol

```diff
@@ -75,6 +76,39 @@ abstract contract BaseLightAccount is BaseAccount, TokenCallbackHandler, UUPSUpg
         }
     }
 
+    /// @notice Creates a contract, this can only be called by this account.
+    /// @param initCode The initCode to deploy. NOTE: This could be replaced with transient storage in the near future,
+    /// depending on gas savings, if any.
+    function create(bytes calldata initCode, uint256 value) external payable virtual onlyAuthorized {
```

Comment:
Do we want these functions to return the deployed address? I know it can be computed separately, but I think the standard create2deployer does this: https://github.com/Arachnid/deterministic-deployment-proxy/blob/be3c5974db5028d502537209329ff2e730ed336c/source/deterministic-deployment-proxy.yul#L14-L15

Reply:
Included!